docs/docusaurus/docs/get_support.md

```diff
@@ -0,0 +1,57 @@
+
+### Search with your favorite browser
+
+It's very likely that someone, somewhere has previously encountered or solved the issue you're currently experiencing. Perform a keyword search in your favorite browser to discover resources outside GX documentation. 
+
+### Try it out
+
+If you're wondering what might happen when you implement a solution, try it out! If you attempt a solution in your development environment, users are unaffected and unaware of what you're up to. Sneaky. 
+
+## Ask the right questions and provide the right information
+
+Taking the time to formulate your questions properly and providing all the relevant background information can save time and improve response times. The following are some recommendations that will help the GX Developer Advocates (DAs) and other community members help you. 
+
+### Include all the relevant information
+
+Clearly explain the issue you're experiencing, or the problem you're trying to solve. When asking questions, you should provide the following:
+
+- The complete error log or logs.
+
+- The relevant code you're trying to run.
+
+- The version of GX OSS you're using.
+
+- A detailed explanation of what you're trying to resolve including what attempts you've made on your own to date.
+
+### Don't post in multiple channels and use threads
+
+When posting questions or asking for help in one of the GX Slack channels, don't post in multiple channels. Instead, select a single, relevant channel and post your message there. When responding to a post, use threads. If you need to post large code blocks, post them in a thread to avoid overwhelming the channel.
+
+### Don't tag individual users
+
+Even if you've previously developed a rapport with a specific GX DA, or a community member, don't tag them in new Slack posts. GX DA and community member availability varies. Also, the GX DAs operate on a rotation schedule to ensure equal time working with the open source community, and you never know who is available or who has the knowledge to solve your issue. Tagging specific users can limit the scope of the response. Also, the community is typically very responsive and helpful. If you don't get a response within a reasonable time, review your post and ensure you've framed your questions correctly and provided the necessary background information. If you don't get a response, or it's unhelpful, try an alternate medium to resolve your issue.
+
+## Select the correct medium for questions and support
+
+GX provides the following resources for questions and support:
+
+- If you've encountered an error and are stuck, get support on the [GX Discourse forum](https://discourse.greatexpectations.io/c/oss-support/11)
```

Comment:
Let's say something here about how we're prioritizing questions in Discourse.

Reply:
@joshzzheng can you clarify your request? How are requests being prioritized?

Reply:
I suggest something like "GX is prioritizing questions in Discourse (over Slack) since the answers are not lost after 90 days. We recommend this channel for quicker response."